### PR TITLE
Fix IsInstalled function

### PR DESCRIPTION
**Pull request by mbussolotto**
## What does this PR change?

see title

## Test coverage
- No tests

- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -51,6 +51,9 @@ linters-settings:
   govet:
     disable:
       - printf
+  #revive:
+  #  rules:
+  #    - name: unused-parameter
```
Comment (mbussolotto) [inline .golangci.yml:56]: @cbosdo I added this commented part: the analysis works as expected...but seems this is not the only case where we're not using a parameter. Let me know if it's fine to push that and fix the other cases late

Comment (cbosdo) [reply .golangci.yml:56]: Last attempts I had with adding a revive rule, the others were not applied. We need good testing of `revive` config to make sure showing the unused parameters don't hide something else.

**Review by cbosdo — APPROVED** [review]

File: .golangci.yml (hunk 1 above)
Comment (cbosdo) [reply .golangci.yml]: I just fixed and enabled the config: using excludes is better as it doesn't disable the other rules.

**Comment by cbosdo** [comment]
> I wonder if all those `_` changes indicates hidden bugs in our code? Like we are supposed to use it but are not using them.

Those remaining in theory are because the function needs to conform to an expected signature. I found a lot of cases where the parameter could simply be removed.

**Comment by cbosdo** [comment]
> I wonder if all those `_` changes indicates hidden bugs in our code? Like we are supposed to use it but are not using them.

I went over the whole diff again and found two of them that could be removed. Basically all the unused parameters in test mocks and in the command entry points have to be turned into `_`. You can see that some of those unused parameters have been simply removed.

**Pull request merged**